packages/flutter/test/widgets/list_wheel_scroll_view_test.dart

```diff
@@ -426,11 +401,11 @@ void main() {
         ),
       );
 
-      final RenderListWheelViewport viewport = tester.firstRenderObject(find.byType(Text)).parent.parent as RenderListWheelViewport;
+      final RenderListWheelViewport? viewport = tester.firstRenderObject(find.byType(Text)).parent!.parent as RenderListWheelViewport?;
```

Comment:
Can it be this? Or is that last `parent` null sometimes? (and below too)
```suggestion
      final RenderListWheelViewport viewport = tester.firstRenderObject(find.byType(Text)).parent!.parent! as RenderListWheelViewport;
```

Reply:
I actually just decided to make the finder code more direct: 
```dart
final RenderListWheelViewport viewport = tester.renderObject(find.byType(ListWheelViewport)) as RenderListWheelViewport;
```